index.bs

```diff
@@ -314,20 +315,27 @@ interface SpeechRecognitionPhraseList {
   <dd>This attribute will set the maximum number of {{SpeechRecognitionAlternative}}s per result.
   The default value is 1.</dd>
 
-  <dt><dfn attribute for=SpeechRecognition>mode</dfn> attribute</dt>
+  <dt><dfn attribute for=SpeechRecognition>options</dfn> attribute</dt>
   <dd>
-   This attribute represents where speech recognition takes place.
-  </dd>
-  <dd>
-    The getter steps are to return the value of {{SpeechRecognition/[[mode]]}}.
-  </dd>
-  <dd>
-    The setter steps are:
-    1. If the {{SpeechRecognitionPhraseList/length}} of {{SpeechRecognition/phrases}} is greater than 0
-        and the system using the given value for {{SpeechRecognition/[[mode]]}} does not support contextual biasing,
-        throw a {{SpeechRecognitionErrorEvent}} with the {{SpeechRecognitionErrorCode/phrases-not-supported}}
-        error code and abort these steps.
-    1. Set {{SpeechRecognition/[[mode]]}} to the given value.
+   This attribute allows specification of a {{SpeechRecognitionOptions}} object to configure options for speech recognition.
+   If this attribute is null, default behaviors apply for these options.
+   The {{SpeechRecognitionOptions}} dictionary has the following members:
+   <dl>
+     <dt><dfn dict-member for="SpeechRecognitionOptions">langs</dfn></dt>
+     <dd>A sequence of {{DOMString}}s. Each {{DOMString}} in the sequence <em class="rfc2119" title="MUST">MUST</em> be a
+     valid [[!BCP47]] language tag. When used with {{SpeechRecognition/start()}}, the user agent will attempt to use the languages in the order provided, selecting the first one that is 
+     available. When used with {{SpeechRecognition/start()}} and {{SpeechRecognition/lang}} is also specified, {{SpeechRecognitionOptions/langs}} <em class="rfc2119" title="MUST">MUST</em> be empty or contain a single language tag identical to {{SpeechRecognition/lang}}.</dd>
+     <dt><dfn dict-member for="SpeechRecognitionOptions">processLocally</dfn></dt>
+     <dd>A boolean that defaults to <code>false</code>.
+     If set to <code>true</code>, it indicates a requirement that the speech recognition process <em class="rfc2119" title="MUST">MUST</em> be performed locally on the user's device, without sending audio data to a remote server.
```

Comment:
I think it would be cleaner to flip the sentence here, not saying that it should not be sent out, but saying it should stay on the device. Here, I can clearly see someone nitpicking and saying it's legal to send the audio to something that isn't a server, which isn't our intent.